### ISSUE TITLE
meteorkitchen seems to be down again

### ISSUE THREAD

**Issue by xauxatz** [perak/kitchen-site#432]
it keeps loading...

**Comment by sazr** [comment]
yes it appears to be down. Hoping to see this web app as it would be useful for me.

**Comment by perak** [comment]
@xauxatz @sazr 

Fixed. 

App is built with meteor 1.6 and looks like it is very unstable - it hangs often. It will be upgraded soon, I hope that will solve this issue.

**Comment by Omar-allahham** [comment]
@perak Could you please update this repo to have the new GUI so we can use it offline?
